### ISSUE TITLE
Time dimension autodetection not aggressive enough?

### ISSUE THREAD

**Issue by jay-hennen** [NOAA-ORR-ERD/gridded#27]
When doing a Variable.from_netCDF, if the variable it finds does not have 'time' in the name it fails to find it and falls back to a constant time.

It should at least look up the dimension directly in the Dataset and try to see if it has a long name that contains 'time' before falling back to a default

**Comment by ChrisBarker-NOAA** [comment]
Indeed, something like this would be good. Do you have an example file that should work that doesn't currently?

**Comment by AmyMacFadyen** [comment]
The HYCOM model output does not load the time correctly. 

        double MT(MT) ;
                MT:long_name = "time" ;
                MT:units = "days since 1900-12-31 00:00:00" ;
                MT:calendar = "standard" ;
                MT:axis = "T" ;
                MT:_CoordinateAxisType = "Time" ;

**Comment by RachaelMueller** [comment]
An initial solution has been pushed today and this issue is covered in a different issue so we are closing this one.
